src/Position.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "lib/openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
+import "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+import "lib/openzeppelin-contracts/contracts/utils/math/SafeMath.sol";
+import "lib/chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import "./BasketHandler.sol";
+import "./PriceConverter.sol";
+import "./Coin.sol";
+
+/**
+ * @dev Position contract manages a tokenized debt position.
+ *
+ * This contract provides the means for an account to manage their debt position
+ * through enforcing adequate collatoralization while withdrawing debt tokens.
+ */
+contract Position is Ownable {
+    using BasketLib for Basket;
+    using PriceConverter for Basket;
+    // uint256 public immutable minRatio;
+    // PriceFeed private immutable priceFeed;
+    Coin private immutable coin;
+    uint256 public debt;
+    bool isInsolvent;
+    Basket collateral;
+    uint256 public constant RATIO = 150;
+    uint256 public constant COLLATERAL_DECIMAL = 1e18;
+    address user;
+    uint256 public constant FEE = 500;
+    uint256 public constant PENALTY = 500;
+    uint256 private s_lastTimeStamp;
```

Comment:
what is `s_lastTimeStamp`? It's hard to infer from the name.

Reply:
lastTimeStamp is a variable that store the time (or block timestamp). This variable will be used in the calculation of accrued interest.